metis-preview/metis-preview-service/src/test/java/eu/europeana/metis/preview/service/ZipServiceTest.java

```diff
@@ -16,24 +25,37 @@
   @Test
   public void readFileToStringList_withOkzipfile_returnsListOfStrings() throws Exception {
       ZipService service = new ZipService();
-      Resource resource = new ClassPathResource("ValidExternalOk.zip");
```

Comment:
Why these changes on the methods of this class were needed?

Reply:
This is because the readFileToStringList method in ZipService has been modified to take a MultipartFile as an input parameter instead of an InputStream.

Reply:
Sure that I understand, but I meant why not use same code as before for getting the resource e.g.:
Resource resource = new ClassPathResource("ValidExternalOk.zip");
resource.getInputStream()

Just seems cleaner like that.
In any case not very important.

Reply:
I agree. I've cleaned up the tests.